solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java

```diff
@@ -141,14 +146,29 @@ public void setId(int id) {
     this.id = id;
   }
 
+  public static Feature fromMap(SolrResourceLoader solrResourceLoader,
+      Map<String,Object> featureMap) {
+    final String type =
```

Comment:
class not type now, should probably also change all the getInstance() calls and params to type for consistency